package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/cordova-acpgriffon",
-  "version": "0.0.1",
```

Comment:
lets move to 1.0.0 ?

Reply:
this is just an update to Griffon SDK, we can keep 0.0.2

I will do 1.0.0 when we rename as Assurance SDK